### ISSUE TITLE
ld.lld: error: version script assignment of 'LINUX_2.6' to symbol '__vdso_sgx_enter_enclave' failed: symbol not defined

### ISSUE THREAD

**Issue by ms178** [ClangBuiltLinux/linux#1769]
clang version 16.0.0 (6a0bbb84cef28ed642a730e55c52447b8c870647) fails to build my custom 6.0.12 Linux Kernel early on with:

```
ld.lld: error: version script assignment of 'LINUX_2.6' to symbol '__vdso_sgx_enter_enclave' failed: symbol not defined
AR      arch/x86/platform/scx200/built-in.a
llvm-objdump: error: 'arch/x86/entry/vdso/vdso64.so.dbg': No such file or directory
```
I haven't seen that one before, any ideas?

**Comment by nathanchance** [comment]
Dupe of #1756. The fix is in -tip, should make 6.1 final: https://git.kernel.org/tip/45be2ad007a9c6bea70249c4cf3e4905afe4caeb

**Comment by ms178** [comment]
Thanks for the hint, Nathan. :)
